content/en/docs/tasks/administer-cluster/highly-available-control-plane.md

```diff
@@ -133,10 +133,18 @@ the [etcd administration guide](https://etcd.io/docs/v2.3/admin_guide/#member-mi
 
 ## Implementation notes
 
-![ha-master-gce](/images/docs/ha-master-gce.png)
+![ha-control-plane](/images/docs/ha-control-plane.svg)
```

Comment:
It would be better to change the path, so that instead of `static/images/docs/ha-control-plane.svg` it is `content/en/docs/SOMETHING`. This is because the image includes text and should be localized.

Reply:
Should we make a `images` folder in `content/en/docs/`, so that it's `content/en/docs/images/ha-control-plane.svg`

Reply:
That works for me.

Reply:
Done!